sdk/nodejs/tests/runtime/testdata/closure-tests/cases/156-Deconstructing-async-function/snapshot.^5.2.0.txt

```diff
@@ -0,0 +1,31 @@
+exports.handler = __f;
+
+function __f0(__0, __1, __2, __3) {
+  return (function() {
+    with({ this: undefined, arguments: undefined }) {
+
+return function (thisArg, _arguments, P, generator) {
+    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f(__0) {
+  return (function() {
+    with({ __awaiter: __f0, f: __f, this: undefined, arguments: undefined }) {
+
+return function /*f*/(_a) {
+    return __awaiter(this, arguments, void 0, function* ({ whatever }) { });
```

Comment:
Before 5.2 the arguments were not passed into `__awaiter` https://github.com/microsoft/TypeScript/pull/56296